views/halloffame.html

```diff
@@ -69,6 +83,81 @@
 			</section >
 			<section class="hof-stats right-side">
 				<h2>My Stats</h2>
+				<div class="stats-helper">
+					<p><span class="regNum"></span> regular HOF conversations</p>
+					<p><span class="gpNum"></span> gold plated conversations</p>
+					<p><span class="gNum"></span> golden conversations</p>
+				</div>
+				<div id="chart"></div>
+			    <script>
+				    var $window = $(window);
+		
+			      	(function(d3) {
+			        'use strict';
+
+				        var dataset = [
+				          { label: 'gold', count: goldNum }, 
+				          { label: 'goldPlated', count: goldPlatedNum },
+				          { label: 'regHof', count: regHofNum }
+				        ];
+
+				        var width = 142;
+				        var height = 142;
+				        var donutWidth = 20;
+
+				        // reize svg for responsive screens;
+				        function checkWidth() {
+					        var windowsize = $window.width();
+					        if (windowsize > 820 && windowsize < 1190) {
+					            width = 90;
+					            height = 90;
+					            donutWidth = 12;
+					            $('#chart').css({
+					            	marginLeft: '1.5rem'
+					            })
+					        } else {
+					        	width = 145	;
+				        		height = 145;
+				        		donutWidth = 20;
+				        		$('#chart').css({
+					            	marginLeft: '-1rem'
+					            })
```

Comment:
It seems like this '#chart' css marginLeft logic could be accomplished with a css media query, rather than JS.  Having it in CSS might also allow us to more easily find and change it later on.  If we decided to change the margin value when it was defined in the JS, we'd have to sift through un-related JS code, while keeping it in the CSS keeps it next to other style-related logic.